pkg/tnf/testcases/base.go

```diff
@@ -176,16 +176,18 @@ const (
 
 // outRegExp types of available regular expression to parse matched result
 var outRegExp = map[RegExType]string{
-	AllowEmpty:     `(.*?)`,
-	AllowAll:       `.+`,
-	EmptyNullFalse: `^\b(null|false)\b$|^$`,
-	NullFalse:      `^\b(null|false)\b$`,
-	True:           `^\b(true)\b$`,
-	Null:           `^\b(null)\b$`,
-	Zero:           `0`,
-	NonZeroNumber:  `^(0*([1-9]\d*)|null)$`,
-	Error:          "error",
-	Digit:          `\d`,
+	// Removing all empty cases as they are not used and will break the parsing
+	// AllowEmpty:     `(.*?)`,
```

Comment:
Just want to make sure, we do still support this case? this is just to get around unit test when terminal simulation is off?

Reply:
Never mind. I thought this was for unit test. Now I see it's for those special combo tests.